### PR TITLE
update kernel

### PR DESCRIPTION
**Pull request by vchiley**
Hey @vishalsubbiah this PR:
 - updt u ctrl to use ctrl process instead of an estimator process.
 - combine v and u ctrl kernels for faster execution

Test: all tests still pass.

PyTorch code updated, reviewed by Atli and merged in [PR 15](https://github.com/Cerebras/online-normalization/pull/15)

### CODE REVIEW

File: online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'  # or any {'0', '1', '2'}
+import tensorflow as tf
+from tensorflow.core.protobuf import config_pb2
+import numpy as np
+try:
+    zero_out_module = tf.load_op_library('./online_norm_gpu.so')
+    modes = ["cpu", "gpu"]
+    # modes = ["gpu"]
```
Comment (vishalsubbiah) [inline online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py]: nit: left over code.

File: online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                            grad_out=inputs,
+                            in_v=tf.cast(inputs[0,:,0], tf.float32),
+                            in_u=tf.cast(inputs[0,:,0], tf.float32),
+                            out=inputs,
+                            scale=inputs[:,:,0],
+                            abkw=0.3,
+                        )
+                    tf.logging.info(f"function: {funcs}, Device: {mode}, dtype: {dtype}")
+                    ans = sess.run(result)
+                    # tf.logging.info(f"found ans: \n {ans}")
```
Comment (vishalsubbiah) [inline online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py]: nit: left over code.

File: online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                results_dict[i] = mode_dict
+        tf.logging.info(f"function: {funcs}, dtype: {dtype}")
+        for k, v in results_dict.items():
+            mode_dict = v
+            if len(modes) == 2:
+                error=np.sum(abs(v["cpu"] - v["gpu"]))
+                tf.logging.info(f"{k}: error:{error}")
+            else:
+                error=v[modes[0]]
+                tf.logging.info(f"{k}: output:{error}")
```
Comment (vishalsubbiah) [inline online-norm/tensorflow/online_norm_tf/csrc/kernel_example_test.py]: nit: new line

**Review by vishalsubbiah — APPROVED** [review]
